### PR TITLE
Changing the CDN to static js files

### PR DESCRIPTION
**Pull request by muneebafzal**

### CODE REVIEW

**Comment by de846** [comment]
Why does this have all of your changed files again? I thought it was merged into develop already?

File: mercury/templates/dashboard.html [hunk 1]
```diff
@@ -60,8 +60,9 @@ <h3>The Simulated Data for each snapshot</h3>
 	<div class="row">
         <div id="articles" style="width:1400px;height:800px;"></div>
     </div>
-	<script src="https://cdn.plot.ly/plotly-latest.min.js"></script>
-	<script src="http://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
+{#	<script src="https://cdn.plot.ly/plotly-latest.min.js"></script>#}
```
Comment (de846) [inline mercury/templates/dashboard.html]:
```suggestion
```

Comment (de846) [reply mercury/templates/dashboard.html]: I would just remove this commented line.

Comment (muneebafzal) [reply mercury/templates/dashboard.html]: okay

**Review by de846 — APPROVED** [review]
👍

**Pull request merged**